index.html

```diff
@@ -159,9 +174,9 @@ <h4 class="dimg">Uber Navigation</h4>
                         </ul>
                     
                     
-                    <button class="buttonlrg">See project</button>
+                    <button class="buttonlrg details-button" id="seeProject4">See project</button>
                 </div>
-                <img src="desktop-img/Snap4.png" alt="project image" class="dimg">
+                <img src="desktop-img/Snap4.png" alt="project image" class="dimg projectImg">
             </section>
```

Comment:
You did well implementing the dynamic view for your `project` section of your portfolio as suggested by the previous reviewer, please I'd kindly suggest you remove lines 106- 180 in your `index.html`  I'm unable to highlight everything so I decided to mention it here since you already implemented this section using dynamic javascript, you should remove the previous HTML code as it's redundant and makes for an unclean code 😄